src/app_service/service/stickers/service.nim

```diff
@@ -0,0 +1,393 @@
+import NimQml, Tables, json, sequtils, chronicles, strutils, atomics, sets, strutils, tables, stint
+import status/types/[sticker, network_type, setting, transaction, network]
```

Comment:
@jrainville don't rely on anything from `status-lib` except methods which do `status-go` API calls. For all types which are currently in the `status-lib` if we need them in the status-desktop part we should add them to appropriate place.